Userland/Libraries/LibVideo/VP9/Parser.h

```diff
@@ -228,19 +242,18 @@ class Parser {
     ReferenceFrame m_comp_fixed_ref;
     ReferenceFrame m_comp_var_ref[2];
     MV m_block_mvs[2][4];
-    u8* m_prev_segment_ids { nullptr };
-
-    u32 m_allocated_dimensions { 0 };
-    bool* m_skips { nullptr };
-    TXSize* m_tx_sizes { nullptr };
-    u32* m_mi_sizes { nullptr };
-    u8* m_y_modes { nullptr };
-    u8* m_segment_ids { nullptr };
-    ReferenceFrame* m_ref_frames { nullptr };
-    InterpolationFilter* m_interp_filters { nullptr };
-    MV* m_mvs { nullptr };
-    MV* m_sub_mvs { nullptr };
-    IntraMode* m_sub_modes { nullptr };
+    Vector<u8> m_prev_segment_ids {};
+
+    Vector<bool> m_skips {};
+    Vector<TXSize> m_tx_sizes {};
+    Vector<u32> m_mi_sizes {};
+    Vector<u8> m_y_modes {};
+    Vector<u8> m_segment_ids {};
+    Vector<Array<ReferenceFrame, 2>> m_ref_frames {};
+    Vector<InterpolationFilter> m_interp_filters {};
+    Vector<Array<MV, 2>> m_mvs {};
+    Vector<Array<Array<MV, 4>, 2>> m_sub_mvs {};
+    Vector<Array<IntraMode, 4>> m_sub_modes {};
```

Comment:
Default constructors don't need the `{}`.
```suggestion
    Vector<Array<IntraMode, 4>> m_sub_modes;
```

Reply:
Fixed.